include/z64environment.h

```diff
@@ -30,6 +30,14 @@ typedef enum {
     /* 13 */ SKYBOX_DMA_PAL2_START
 } SkyboxDmaState;
 
+typedef enum {
+    /* 0 */ SANDSTORM_OFF,
+    /* 1 */ SANDSTORM_FILL,
+    /* 2 */ SANDSTORM_UNFILL,
```

Comment:
the fill part is referring specifically to the screen fill portion which blocks the whole screen. its not intended to mean the sandstorm as a whole filling the screen. Theres a solid part of the sandstorm that exits while the rest of the sandstorm stays around

Reply:
It's fine as it is imo, afaict the sandstorm drawing works the same regardless of the sandstorm state (same dlist), it's just different prim and env rgba values.
The fancy combiner `gFieldSandstormDL` uses does the rest

The names are good as is already, and it will also be better eventually :tm: when the `Environment_DrawSandstorm` variables are better documented